README.md

```diff
@@ -1,3 +1,7 @@
+> [!WARNING]
+> This package will become redundant with WordPress 6.8+ as bcrypt becomes the default password hashing method.
+> See [WordPress 6.8 will use bcrypt for password hashing](https://make.wordpress.org/core/2025/02/17/wordpress-6-8-will-use-bcrypt-for-password-hashing/) for more details.
```

Comment:
Should we add a note about the migration process? eg: just uninstall, existing passwords should work without issue

Reply:
Good idea, will update when I’m on my machine later